experimental/LieAlgebras/src/CoxeterGroup.jl

```diff
@@ -0,0 +1,3 @@
+abstract type CoxeterGroup end # <: Group
+
+#function is_coxeter_matrix(M::ZZMatrix) end
```

Comment:
Meta-remark: maybe at some point in the future rename the `LieAlgebras` directory to `LieTheory` or `AlgebraicLieTheory` (of course only after all big PRs have been merged, i.e., this is far from urgent)